integration-cli/docker_cli_rmi_test.go

```diff
@@ -297,10 +232,9 @@ func (s *DockerSuite) TestRmiContainerImageNotFound(c *check.C) {
 
 	// Try to remove the image of the running container and see if it fails as expected.
 	out, _, err := dockerCmdWithError("rmi", "-f", imageIds[0])
-	if err == nil || !strings.Contains(out, "image is being used by running container") {
-		c.Log(out)
-		c.Fatal("The image of the running container should not be removed.")
-	}
+	// The image of the running container should not be removed.
+	c.Assert(err, checker.NotNil)
```

Comment:
Could you move the comment in the end of the line or in a `check.Commentf(…)` so we don't loose the "context" when this assertion fails ?